Source/WebCore/platform/graphics/avfoundation/cf/CDMSessionAVFoundationCF.h

```diff
@@ -41,9 +41,11 @@ class MediaPlayerPrivateAVFoundationCF;
 class CDMSessionAVFoundationCF final : public LegacyCDMSession {
     WTF_MAKE_TZONE_ALLOCATED(CDMSessionAVFoundationCF);
 public:
-    CDMSessionAVFoundationCF(MediaPlayerPrivateAVFoundationCF& parent, LegacyCDMSessionClient&);
+    static Ref<CDMSessionAVFoundationCF> create(MediaPlayerPrivateAVFoundationCF& parent, LegacyCDMSessionClient&);
```

Comment:
Pretty concerned about the `MediaPlayerPrivateAVFoundationCF& m_parent;` now that we're ref-counted. Maybe this subclass could forward the ref-counting to its parent? (didn't check if feasible).

Reply:
Deleted this code.